components/library/fftw-3/fftw-3.p5m

```diff
@@ -24,40 +25,44 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file files/libfftw3.3.sunman path=usr/share/man/man3/libfftw3.3
 
-# We build fftw float versions separately and don't 
-# install them, so we have to pick up produced files 
-# manually from build dirs and create necessary links
```

Comment:
Have you checked that float version doesn't overwrite files of non-float version?

Reply:
Would these checks be enough?
```
{global} newman@lenovo:~ $ ls -l /usr/lib/{32,64}/libfftw3*.so.3.5.8 
-r-xr-xr-x   1 root     bin       969192 Jun 30 08:41 /usr/lib/32/libfftw3.so.3.5.8*
-r-xr-xr-x   1 root     bin       941108 Jun 30 08:41 /usr/lib/32/libfftw3f.so.3.5.8*
-r-xr-xr-x   1 root     bin      1294848 Jun 30 08:41 /usr/lib/64/libfftw3.so.3.5.8*
-r-xr-xr-x   1 root     bin      1254528 Jun 30 08:41 /usr/lib/64/libfftw3f.so.3.5.8*

{global} newman@lenovo:~ $ md5sum /usr/lib/{32,64}/libfftw3*.so.3.5.8 
004c442854a6c4718c9cf028618fb5c5  /usr/lib/32/libfftw3.so.3.5.8
bcc18bb81e917ebc67ed27a7de296ca2  /usr/lib/32/libfftw3f.so.3.5.8
7ec0f7c61ab0a4f1dd4720261e710de2  /usr/lib/64/libfftw3.so.3.5.8
d2d2288848dc661246ddd0d5b199b866  /usr/lib/64/libfftw3f.so.3.5.8

{global} newman@lenovo:~ $ for i in /usr/lib/{32,64}/libfftw3*.so.3.5.8 ; do echo === $i; readelf -a -W $i | grep fftw.*_dimcmp$ | head -n1; done
=== /usr/lib/32/libfftw3.so.3.5.8
000e3104  0000d106 R_386_GLOB_DAT         000196c0   fftw_dimcmp
=== /usr/lib/32/libfftw3f.so.3.5.8
000dc104  0002b206 R_386_GLOB_DAT         00019cb0   fftwf_dimcmp
=== /usr/lib/64/libfftw3.so.3.5.8
   272: 0000000000034ed0   194 FUNC    GLOBAL DEFAULT   15 fftw_dimcmp
=== /usr/lib/64/libfftw3f.so.3.5.8
   581: 0000000000035220   194 FUNC    GLOBAL DEFAULT   15 fftwf_dimcmp
```

So much for libraries. The rest (.pc, .cmake, man and info pages) I checked manually and the content looked sane to me.